### ISSUE TITLE
1.14 Wall banner not recognized as block type

### ISSUE THREAD

**Issue by Ydda** [SkriptLang/Skript#2080]
```
on right click:
	if clicked block is wall banner:
		broadcast "a"
on place:
	broadcast "%event-block%"
```
The on place -event returns a name for wall banner, but the right click event returns an error saying wall banner is not a valid block type.

**Comment by TheBentoBox** [comment]
This is because there is no grouped alias for wall/standing banners. It currently requires a color to be specified so this would be a good addition.

**Comment by TheBentoBox** [comment]
I've added grouped aliases for the banners and they should be included as of 2.4-alpha2.
